### PR TITLE
Federation fixes.

### PR DESCRIPTION
**Pull request by erikjohnston**

### CODE REVIEW

**Comment by erikjohnston** [comment]
Jenkins: Test this please

**Comment by erikjohnston** [comment]
Or.... test this please?

File: synapse/storage/__init__.py [hunk 1]
```diff
@@ -128,16 +128,21 @@ def persist_event(self, event, context, backfilled=False,
             pass
 
     @defer.inlineCallbacks
-    def get_event(self, event_id, allow_none=False):
-        events = yield self._get_events([event_id])
+    def get_event(self, event_id, check_redacted=True,
+                  get_prev_content=False, allow_rejected=False,
+                  allow_none=False):
```
Comment (NegativeMjark) [inline synapse/storage/__init__.py]: Might want to add a bit of __doc__ string for the get_event now that it has a few parameters.

**Comment by erikjohnston** [comment]
Can someone have a look at this again please?  It passes sytest now :)

**Pull request merged**